app_dart/lib/src/model/gerrit/commit.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+
+import 'package:json_annotation/json_annotation.dart';
+
+import '../../request_handling/body.dart';
+
+part 'commit.g.dart';
+
+@JsonSerializable()
+class GerritCommit extends JsonBody {
```

Comment:
Nit: docs/links to gerrit proto defination would be helpful. Same for other data models.

Reply:
Done